### PR TITLE
Translate sphinx.po

### PR DESCRIPTION
**Pull request by StevenHsuYL**
This is a small piece at Doc homepage.
The version's notes in index sidebar ("Docs by version" column) were not updated, so the translation would be incorrect.

### CODE REVIEW

File: sphinx.po [hunk 1]
```diff
@@ -72,222 +74,222 @@ msgstr "由此開始閱讀"
...
 
 #: ../../tools/templates/indexsidebar.html:8
 msgid "Python 3.7 (stable)"
-msgstr ""
+msgstr "Python 3.7（穩定）"
 
 #: ../../tools/templates/indexsidebar.html:9
 msgid "Python 3.6 (security-fixes)"
-msgstr ""
+msgstr "Python 3.8（安全性修護）"
```
Comment (adrianliaw) [inline sphinx.po]: s/3.8/3.6

**Review by adrianliaw — COMMENTED** [review]
讚爆

**Comment by StevenHsuYL** [comment]
Fixed.
Can you make html of this page? I tried several times but always failed...

File: sphinx.po [hunk 2]
```diff
@@ -18,39 +19,40 @@ msgstr ""
...
 
 #: ../../tools/templates/dummy.html:8
 msgid "Deprecated since version {deprecated}, removed in version {removed}"
-msgstr ""
+msgstr "自從版本 {deprecated} 後不推薦使用，已從版本 {removed} 中移除。"
 
 #: ../../tools/templates/indexcontent.html:8
 msgid "Welcome! This is the documentation for Python %(release)s."
-msgstr ""
+msgstr "歡迎！這是 Python %(release) 的說明文件。"
```
Comment (adrianliaw) [inline sphinx.po]: 可能是因為這裡少了 s

**Comment by StevenHsuYL** [comment]
It works after fixing "s". Thanks!
I use the version's note according to the sidebar of online page ( https://docs.python.org/3.9/ ). 
But it has some problems at this version of sphinx.po
I would leave this issue to you. :p

**Pull request merged**